NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<configuration>
+<packageSources>
+ <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+ <add key="Roslyn nightly builds" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
Regardless how to distribute, we still ned to bring in the 3.0.0 dll's from somewhere before we create the NuGet package?